app/javascript/dashboard/routes/dashboard/settings/assignmentPolicy/pages/AgentAssignmentEditPage.vue

```diff
@@ -0,0 +1,351 @@
+<script setup>
+import { computed, reactive, watch, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useStore, useMapGetter } from 'dashboard/composables/store';
+import { useRoute, useRouter } from 'vue-router';
+import { useAlert } from 'dashboard/composables';
+import { useConfig } from 'dashboard/composables/useConfig';
+import { getInboxIconByType } from 'dashboard/helper/inbox';
+
+import Breadcrumb from 'dashboard/components-next/breadcrumb/Breadcrumb.vue';
+import Button from 'dashboard/components-next/button/Button.vue';
+import SettingsLayout from 'dashboard/routes/dashboard/settings/SettingsLayout.vue';
+import BaseInfo from 'dashboard/components-next/AssignmentPolicy/components/BaseInfo.vue';
+import RadioCard from 'dashboard/components-next/AssignmentPolicy/components/RadioCard.vue';
+import FairDistribution from 'dashboard/components-next/AssignmentPolicy/components/FairDistribution.vue';
+import DataTable from 'dashboard/components-next/AssignmentPolicy/components/DataTable.vue';
+import AddDataDropdown from 'dashboard/components-next/AssignmentPolicy/components/AddDataDropdown.vue';
+import ConfirmInboxDialog from 'dashboard/routes/dashboard/settings/assignmentPolicy/pages/components/ConfirmInboxDialog.vue';
+
+const OPTIONS = {
+  ORDER: ['round_robin', 'balanced'],
+  PRIORITY: ['earliest_created', 'longest_waiting'],
```

Comment:
Let's create a single constants file and use it across. See this [comment](https://github.com/chatwoot/chatwoot/pull/12400/files#r2335885117)